css/properties.json

```diff
@@ -6472,6 +6472,30 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "place-items": {
+    "syntax": "<'align-items'> <'justify-items'>?",
+    "media": "visual",
+    "inherited": false,
+    "animationType": [
```

Comment:
One question: Should I use an array like this for the shorthand property, and if so why does `place-content` not use arrays?

Reply:
If it shouldn't use an array, here's the format I used initially (including this for my future reference if it's ever necessary):
```json
"animationType": "discrete",
"initial": "normal legacy",
"computed": "asSpecified"
```

Reply:
I'm not sure, and I wish the guidance was clearer here, but I think it's because [the specification for the property](https://drafts.csswg.org/css-align-3/#place-items-property) has:

Animatable: | no
-- | --

...so there's no value in saying "as each of the properties of the shorthand:", it's more direct and simpler just to say "no" (or "discrete" as we put it, which seems confusing but is perhaps driven by the issue linked from https://drafts.csswg.org/css-transitions/#animatable-properties).

Does that sound plausible to you?

Reply:
Sounds reasonable to me, I've updated the PR.